Doc/library/socketserver.rst

```diff
@@ -116,23 +116,23 @@ server is the address family.
    :class:`ForkingMixIn` and the Forking classes mentioned below are
    only available on POSIX platforms that support :func:`~os.fork`.
 
-   :meth:`socketserver.ForkingMixIn.server_close` waits until all child
+   :meth:`!ForkingMixIn.server_close` waits until all child
```

Comment:
Should all these be documented rather than excluding via `!`?

Reply:
They are documented, but in a base class. I don't think that we should document implementation methods, they are just, well, implementations :)

Reply:
https://github.com/python/cpython/blob/777f882141dd001beab31fccec3d227ac60c7031/Doc/library/socketserver.rst?plain=1#L251

Reply:
Verbose, but a potential option?
```suggestion
   :meth:`ForkingMixIn.server_close <BaseServer.server_close>` waits until all child
```

I think it might be worth documenting these methods, though, as their behaviour does differ significantly.

A

Reply:
I like the verbose option :)

I don't agree that their behavior is different: they all just clean things up afterwards. But in a different way (which is an implementation detail that should not be a part of the docs).